test/parallel/test-tls-ocsp-callback.js

```diff
@@ -59,82 +58,56 @@ function test(testOptions, cb) {
     options.passphrase = testOptions.passphrase;
   }
 
-  const server = tls.createServer(options, function(cleartext) {
+  const server = tls.createServer(options, common.mustCall((cleartext) => {
     cleartext.on('error', function(er) {
       // We're ok with getting ECONNRESET in this test, but it's
       // timing-dependent, and thus unreliable. Any other errors
       // are just failures, though.
       if (er.code !== 'ECONNRESET')
         throw er;
     });
-    ++requestCount;
     cleartext.end();
-  });
-  server.on('OCSPRequest', function(cert, issuer, callback) {
-    ++ocspCount;
-    assert.ok(Buffer.isBuffer(cert));
-    assert.ok(Buffer.isBuffer(issuer));
-
-    // Just to check that async really works there
-    setTimeout(function() {
-      callback(null,
-               testOptions.response ? Buffer.from(testOptions.response) : null);
-    }, 100);
-  });
+  }, requestCount));
+
+  if (!testOptions.ocsp)
+    server.on('OCSPRequest', common.mustNotCall());
+  else
+    server.on('OCSPRequest', common.mustCall((cert, issuer, callback) => {
+      assert.ok(Buffer.isBuffer(cert));
+      assert.ok(Buffer.isBuffer(issuer));
+
+      // Callback a little later to ensure that async really works.
+      return setTimeout(callback, 100, null, testOptions.response ?
+                        Buffer.from(testOptions.response) : null);
+    }));
+
   server.listen(0, function() {
     const client = tls.connect({
       port: this.address().port,
-      requestOCSP: testOptions.ocsp !== false,
-      secureOptions: testOptions.ocsp === false ?
-        SSL_OP_NO_TICKET : 0,
+      requestOCSP: testOptions.ocsp,
+      secureOptions: testOptions.ocsp ? 0 : SSL_OP_NO_TICKET,
       rejectUnauthorized: false
-    }, function() {
-      clientSecure++;
-    });
-    client.on('OCSPResponse', function(resp) {
-      ocspResponse = resp;
-      if (resp)
-        client.destroy();
-    });
-    client.on('close', function() {
+    }, common.mustCall(() => { }, requestCount));
+
+    client.on('OCSPResponse', common.mustCall((resp) => {
+      if (testOptions.response) {
+	assert.strictEqual(resp.toString(), testOptions.response);
+	client.destroy();
```

Comment:
Indentation is off here.

Reply:
Thanks. I tried to improve my indentation settings, and they started injecting TAB. Oops.